### PR TITLE
Highlighting the placeholder sections of dev and post submission dashboard

### PR DESCRIPTION
**Pull request by hammerfunk**
Also fixed some visual bugs that I found

### CODE REVIEW

File: app/assets/stylesheets/admin/page-applications.scss [hunk 1]
```diff
@@ -238,6 +247,10 @@
     margin-bottom: 5px;
     font-size: 15px;
   }
+
+  p.p-lg {
```
Comment (matthewford) [inline app/assets/stylesheets/admin/page-applications.scss]: Merge rule `p.p-lg` with rule on line 246<br>Avoid qualifying class selectors with an element.

File: app/assets/stylesheets/application-admin.scss [hunk 2]
```diff
@@ -23,3 +23,27 @@ $button-colour: #00823b;
...
+
+// HIGHLIGHT PLACEHOLDER BITS
+.todo-note {
+  display: none;
+}
+
+.layout-dev {
+  .todo-note {
+    display: block;
+    background: #ffff00;
```
Comment (matthewford) [inline app/assets/stylesheets/application-admin.scss]: Color literals like `#ffff00` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/application-admin.scss [hunk 3]
```diff
@@ -23,3 +23,27 @@ $button-colour: #00823b;
...
+    color: #333;
```
Comment (matthewford) [inline app/assets/stylesheets/application-admin.scss]: Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/application-admin.scss [hunk 4]
```diff
@@ -23,3 +23,27 @@ $button-colour: #00823b;
...
+    font-weight: bold;
+  }
+
+  .todo-placeholder {
+    outline: 4px solid #ffff00 !important;
```
Comment (matthewford) [inline app/assets/stylesheets/application-admin.scss]: Color literals like `#ffff00` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used

File: app/assets/stylesheets/application-admin.scss [hunk 5]
```diff
@@ -23,3 +23,27 @@ $button-colour: #00823b;
...
+  }
+
+  .todo-unconfirmed {
+    outline: 4px solid #66ff66 !important;
```
Comment (matthewford) [inline app/assets/stylesheets/application-admin.scss]: Color literals like `#66ff66` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used

File: app/assets/stylesheets/qae.scss [hunk 6]
```diff
@@ -52,3 +52,27 @@ $gutter-one-sixth: $gutter/6;
...
+
+// HIGHLIGHT PLACEHOLDER BITS
+.todo-note {
+  display: none;
+}
+
+.layout-dev {
+  .todo-note {
+    display: block;
+    background: #ffff00;
```
Comment (matthewford) [inline app/assets/stylesheets/qae.scss]: Color literals like `#ffff00` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/qae.scss [hunk 7]
```diff
@@ -52,3 +52,27 @@ $gutter-one-sixth: $gutter/6;
...
+    color: #333;
```
Comment (matthewford) [inline app/assets/stylesheets/qae.scss]: Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.

File: app/assets/stylesheets/qae.scss [hunk 8]
```diff
@@ -52,3 +52,27 @@ $gutter-one-sixth: $gutter/6;
...
+    font-weight: bold;
+  }
+
+  .todo-placeholder {
+    outline: 4px solid #ffff00 !important;
```
Comment (matthewford) [inline app/assets/stylesheets/qae.scss]: Color literals like `#ffff00` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used

File: app/assets/stylesheets/qae.scss [hunk 9]
```diff
@@ -52,3 +52,27 @@ $gutter-one-sixth: $gutter/6;
...
+  }
+
+  .todo-unconfirmed {
+    outline: 4px solid #66ff66 !important;
```
Comment (matthewford) [inline app/assets/stylesheets/qae.scss]: Color literals like `#66ff66` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used

File: app/assets/stylesheets/application-admin.scss [hunk 10]
```diff
@@ -23,3 +23,31 @@ $button-colour: #00823b;
...
+    color: $todo-black;
+    font-weight: bold;
+  }
+
+  .todo-placeholder {
+    outline: 4px solid $todo-yellow !important;
+  }
+
+  .todo-unconfirmed {
+    outline: 4px solid $todo-green !important;
```
Comment (matthewford) [inline app/assets/stylesheets/application-admin.scss]: !important should not be used

File: app/assets/stylesheets/qae.scss [hunk 11]
```diff
@@ -52,3 +52,31 @@ $gutter-one-sixth: $gutter/6;
...
+    color: $todo-black;
+    font-weight: bold;
+  }
+
+  .todo-placeholder {
+    outline: 4px solid $todo-yellow !important;
+  }
+
+  .todo-unconfirmed {
+    outline: 4px solid $todo-green !important;
```
Comment (matthewford) [inline app/assets/stylesheets/qae.scss]: !important should not be used

File: app/assets/stylesheets/application-admin.scss (hunk 10 above)
Comment (hammerfunk) [inline app/assets/stylesheets/application-admin.scss]: ignore this is temporary

File: app/assets/stylesheets/qae.scss (hunk 11 above)
Comment (hammerfunk) [inline app/assets/stylesheets/qae.scss]: ignore this is temporary

**Comment by RuslanHamidullin** [comment]
Good :+1: Awaitinig for travis results

**Pull request merged**